common/_variables.scss

```
@@ -3,6 +3,10 @@ $main-color: #ff8000 !default;
 $link-color: #126f9a !default;
 $header-bg: #fff !default;
 $footer-bg: #fff !default;
+$header-btn-bg: $header-bg !default;
```

Nit: a white space between the other variables and header variables for better readability
```suggestion

$header-btn-bg: $header-bg !default;
```